packages/features/ee/video/ViewRecordingsDialog.tsx

```diff
@@ -157,7 +157,7 @@ export const ViewRecordingsDialog = (props: IViewRecordingsDialog) => {
   const { t, i18n } = useLocale();
   const { isOpenDialog, setIsOpenDialog, booking, timeFormat } = props;
 
-  const { hasPaidPlan, isLoading: isTeamPlanStatusLoading } = useHasPaidPlan();
+  const { hasTeamPlan, isLoading: isTeamPlanStatusLoading } = useHasTeamPlan();
```

Comment:
useHasPaidPlan hook checks for premium username or active team member but we only want to check if user is part of an active team